Userland/Utilities/zip.cpp

```diff
@@ -53,17 +53,17 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     Archive::ZipOutputStream zip_stream { file_stream };
 
-    auto add_file = [&](DeprecatedString path) {
+    auto add_file = [&](DeprecatedString path) -> ErrorOr<void> {
```

Comment:
This commit should probably be LibArchive+Utilities and mention in the commit message that there's some TRY-related cleanup of unzip, zip and tar thrown in.